trieste/acquisition/optimizer.py

```diff
@@ -261,22 +345,39 @@ def _perform_parallel_continuous_optimization(
     which has equal upper and lower bounds, i.e. we specify an equality constraint
     for this dimension in the scipy optimizer.
 
-    :param target_func: The function to maximise, with input shape [..., 1, D] and
-        output shape [..., 1].
+    This function also support the maximization of vectorized target functions (with
+    vectorization V).
+
+    :param target_func: The function(s) to maximise, with input shape [..., V, D] and
+        output shape [..., V].
     :param space: The original search space.
-    :param starting_points: The points at which to begin our optimizations. The
-        leading dimension of `starting_points` controls the number of individual
-        optimization runs.
+    :param starting_points: The points at which to begin our optimizations of shape
+        [num_optimization_runs, V, D]. The leading dimension of
+        `starting_points` controls the number of individual optimization runs
+        for each of the V target functions.
     :param optimizer_args: Keyword arguments to pass to the Scipy optimizer.
```

Comment:
what if somebody passes a different method, does it overwrites bfgs one set in `ScipyLbfgsBGreenlet` - perhaps worth pointing out method, jac and bounds cannot/shouldn't be changed

Reply:
good point